README.md

```diff
@@ -54,10 +55,9 @@ You will also need function to apply JSON Patch, we suggest [fast JSON Patch](ht
 Name      | Arguments                     | Default | Description
 ---       | ---                           | ---     | ---
 `send`    | *JSONPatch* sequence          |         | Changes given JSON Patch to Versioned JSON Patch
-`receive` |                               |         | Receives, and eventually applies given Versioned JSON Patch, to the object
-          | *Object* obj                  |         | object to be changed
-          | *VersionedJSONPatch* sequence |         | Versioned JSON Patch to be queued and applied
-`JSONPatchQueue`  | *Function* transform |         |  `function(pacth, againstPatch_es)` function to transform given JSON Patch agains others
+`receive` | *VersionedJSONPatch* sequence |         | Versioned JSON Patch to be queued and applied
+`JSONPatchQueue`  | *Object* obj          |         | Target object where patches are applied
+`JSONPatchQueue`  | *Function* transform |         |  `function(patch, againstPatch_es)` function to transform given JSON Patch against others
```

Comment:
Formatting of a table is corrupted for some reason.
![obraz](https://user-images.githubusercontent.com/17435/31120515-f6a85400-a834-11e7-92b3-ddf335f5c5c6.png)

Maybe indentation interpretted as code block?

 Also, I was not putting anything into the first column, if the row contains another parameter of the same function.

Reply:
That's weird. It is rendered ok in Atom, which is Githubs -_- I'll see.